test/passing/attributes.ml

```diff
@@ -275,12 +277,14 @@ let _ = f ((0, 0) [@a])
 
 ;;
 ( a________________________________________
-, b________________________________________ ) [@a]
+, b________________________________________ )
+[@a]
```

Comment:
This attribute, and the two below it, definitely look like they should fit.

Reply:
The phrase that the attribute attaches to itself breaks. So breaking in this case is consistent with analogous situations.